### ISSUE TITLE
Add a license file

### ISSUE THREAD

**Issue by karan** [someones/jaweson#1]
Without a license, this project isn't open source and no one can use the code.

Please use something like http://choosealicense.com/ to decide what license to use. I recommend MIT or GPL.

**Comment by adamlwgriffiths** [comment]
Not true, by default on github if no license is provided the code is essentially public domain.
http://stackoverflow.com/a/4007713/1591957

setup.py has the license set as 'BSD'
https://github.com/someones/jaweson/blob/master/setup.py

I'll add a LICENSE file to be more explicit tomorrow.
